src/timer/SDL_timer.c

```diff
@@ -386,4 +388,40 @@ SDL_RemoveTimer(SDL_TimerID id)
     return canceled;
 }
 
+#else
+
+#include <emscripten/emscripten.h>
+
+int
+SDL_TimerInit(void)
+{
+    return 0;
+}
+
+void
+SDL_TimerQuit(void)
+{
+}
+
+SDL_TimerID
+SDL_AddTimer(Uint32 interval, SDL_TimerCallback callback, void *param)
+{
+    return EM_ASM_INT({
+        return window.setTimeout(function() {
```

Comment:
better to call `Browser.safeSetTimeout`, which will ensure the timeout is cancelled if the runtime was shut down.